app/commands.py

```diff
@@ -914,3 +925,32 @@ def populate_annual_billing_with_defaults(year, missing_services_only):
 
     for service in active_services:
         set_default_free_allowance_for_service(service, year)
+
+
+@click.option('-u', '--user-id', required=True)
+@notify_command(name='local-dev-broadcast-permissions')
+def local_dev_broadcast_permissions(user_id):
+    if os.getenv('NOTIFY_ENVIRONMENT', '') not in ['development', 'test']:
+        current_app.logger.error('Can only be run in development')
+        return
+
+    user = User.query.filter_by(id=user_id).one()
+
+    user_broadcast_services = Service.query.filter(
+        Service.permissions.any(permission='broadcast'),
+        Service.users.any(id=user_id)
```

Comment:
You could consider another filter to ensure that only active services are changed